src/components/check-status-responses/CheckStatusShippingFedex.tsx

```diff
@@ -33,6 +33,7 @@ export const CheckStatusShippingFedex: FC<CheckStatusShippingFedexProps> = ({
             </p>
             <p>
               <ExternalLink
+                data-gc-analytics-exempt=""
```

Comment:
See comment above.

Reply:
Fixed for you personal pleasure. You can need eat a big bag of chips.